contracts/lib/Verifiers.sol

```diff
@@ -116,14 +116,16 @@ contract Verifiers is Assertions, SignatureVerification {
             return false;
         }
 
+        uint256 orderStatusNumerator = orderStatus.numerator;
+
         // If the order is not entirely unused...
-        if (orderStatus.numerator != 0) {
+        if (orderStatusNumerator != 0) {
             // ensure the order has not been partially filled when not allowed.
             if (onlyAllowUnused) {
                 // Always revert on partial fills when onlyAllowUnused is true.
                 revert OrderPartiallyFilled(orderHash);
                 // Otherwise, ensure that order has not been entirely filled.
```

Comment:
Unrelated to your PR, but could you also move this comment to below the else statement?

Reply:
Wasn't sure which comment you were referring to here.